package.json

```diff
@@ -10,6 +10,7 @@
   "scripts": {
     "codegen": "lerna exec --parallel --scope @mstable/*-subgraph -- yarn run codegen",
     "build": "lerna exec --parallel --scope @mstable/*-subgraph -- yarn run build",
+    "prepare:config": "lerna exec --parallel --scope @mstable/*-subgraph -- yarn run prepare:config",
```

Comment:
Why do we need this? `prepare:config` calls all the other `prepare:config` which calls all the `prepare:mainnet`.
Should i then not just use `prepare:mainnet` that already does this?

Reply:
We need this because some subgraphs are polygon only, looking at all the configs we have subgraphs in the following categories:
- mainnet only
- mainnet and polygon
- polygon only

Reply:
@dimsome check `staking-rewards` subragraph

Reply:
Ah ok, I understand. The config part doesn't really convey the right meaning.

Should the script just be `prepare` and that does it for mainnet and polygon?
Then the others can be more specific.

Reply:
`prepare` is something else https://docs.npmjs.com/cli/v8/using-npm/scripts#prepare-and-prepublish